angular/src/app/player-character/model.ts

```diff
@@ -1,54 +1,54 @@
 export enum Alignment {
-  "Lawful Evil",
-  "Lawful Neutral",
-  "Lawful Good",
-  "Neutral Good",
-  "Neutral Neutral",
-  "Neutral Evil",
-  "Chaotic Good",
-  "Chaotic Neutral",
-  "Chaotic Evil"
+  le = "Lawful Evil",
+  ln = "Lawful Neutral",
+  lg = "Lawful Good",
+  ng = "Neutral Good",
+  nn = "Neutral Neutral",
+  ne = "Neutral Evil",
+  cg = "Chaotic Good",
+  cn = "Chaotic Neutral",
+  ce = "Chaotic Evil"
 }
 
 export interface PlayerCharacter {
 
   /*** Database Properties ***/
   _id: string;
-  _rev: string;
+  _rev?: string;
 
   /*** Attribute Properties ***/
-  age: number;
-  alignment: Alignment;
-  experience: number;
-  handed: 'RIGHT'|'LEFT';
-  name: string;
-  gender: 'MALE'|'FEMALE';
-  hair: string;
-  eyes: string;
-  height: number;
-  weight: number;
-  player: string;
-  notes: any;
+  age?: number;
+  alignment?: Alignment;
+  experience?: number;
+  handed?: 'RIGHT'|'LEFT';
+  name?: string;
+  gender?: 'MALE'|'FEMALE';
```

Comment:
Are you assuming their gender?

Reply:
That's an enum. It can be extended in the future, or be made into a textbox at your leisure.